apps/src/templates/SchoolNotFound.jsx

```diff
@@ -156,6 +161,18 @@ export default class SchoolNotFound extends Component {
       <div style={styles.errors}>{i18n.schoolInfoRequired()}</div>
     );
 
+    // Check if country selected is US or non-US.  This new check
+    // is used to decide which fields in the school info interstitial
+    // form should be required.
+    let countryIsNotUS = false;
```

Comment:
Should this be `countryIsUS`?

Either way, it's best to avoid negatives in variable names/definitions if possible. So even if it should be `countryIsNotUS`, it would be great to refactor this so that it is `countryIsUS` if it's not too much work.

Reply:
Seconded, it looks like we mean the opposite here.